ecl/hql/hqlfold.cpp

```diff
@@ -2154,6 +2154,82 @@ IHqlExpression * foldConstantOperator(IHqlExpression * expr, unsigned foldOption
             }
             break;
         }
+    case no_regex_findset:
+        {
+            IValue * t0 = expr->queryChild(0)->queryValue();
+            IValue * t1 = expr->queryChild(1)->queryValue();
+            IHqlExpression * c2 = queryRealChild(expr, 2);
+            IValue * t2 = c2 ? c2->queryValue() : NULL;
```

Comment:
There isn't a form of REGEXFINDSET with a third expression parameter - so this is redundant.